src/resources/views/crud/fields/repeatable.blade.php

```diff
@@ -4,6 +4,8 @@
   $field['value'] = old($field['name']) ? old($field['name']) : (isset($field['value']) ? $field['value'] : (isset($field['default']) ? $field['default'] : '' ));
   // make sure the value is a JSON string (not array, if it's cast in the model)
   $field['value'] = is_array($field['value']) ? json_encode($field['value']) : $field['value'];
+  $field['init_rows'] = $field['init_rows'] ?? 1;
+  $field['max_rows'] = $field['max_rows'] ?? 0;
```

Comment:
@pxpm what do you think about renaming these to `min` and `max`?

I agree that `init_rows` and `max_rows` are more intuitive. Totally. But [the table field](https://backpackforlaravel.com/docs/4.1/crud-fields#table) has basically the same functionality, using `min` and `max`. 

In this case I'm wondering if it isn't better to be consistent than explicit...

If we do change these please re-request Antonio's review too.

Reply:
Hello @tabacitu , I would rather change the `table` ones, as I think `init_rows` and `max_rows` are way more readable and accurate in the meaning than `max` and `min`. 

Also `min` is not accounted here. This PR only takes into account the number of initial rows, and the maximum allowed rows. 

I think we should add the `min` rows as a separate PR ?

Reply:
Pff... Yeah I know... But if we change the `table` field then we'd either be:
(a) introducing a breaking change to it
(b) having two syntaxes for the same thing, for the same field (if we use the current `min` and `max` as fallbacks to avoid it being a breaking change)

Which is... messy.

I say for consistency's sake:
- let's rename to `min` and `max`
- let's add the minimum functionality (with the init being the minimum - I think that's intuitive)

That way if you've used the `table` field it'll be intuitive to use the same functionality inside the `repeatable` field. I think that's the way backpackers see it - that we just "_added the same functionality to the `repeatable` field_".

Then in v4.2 we _could_ rename `min` and `max` to `min_rows` and `max_rows`, for both fields - either as a breaking change or with a fallback. Please add a new issue for that so we don't forget.

Reply:
Ok, renaming would not be troube here. 

Problem for me now becomes assuming that `init_rows` equals `min_rows`. It might not always be true, I can init `2` rows, but minimum I accept `1`.

So `min`, `max`, `init_rows` ?